packages/browser-data-collector/src/collectors/performance-timing.ts

```diff
@@ -33,31 +33,34 @@ const normalize = ( value: number, start: number ): number => {
 
 export const collector: Collector = ( report ) => {
 	// Ensure these values are relative to the beginning of the report
-	report.data.set( 'navigationStart', normalize( getNavigationStart(), report.start ) );
-	report.data.set( 'unloadEventStart', normalize( getUnloadEventStart(), report.start ) );
-	report.data.set( 'unloadEventEnd', normalize( getUunloadEventEnd(), report.start ) );
-	report.data.set( 'redirectStart', normalize( getRedirectStart(), report.start ) );
-	report.data.set( 'redirectEnd', normalize( getRedirectEnd(), report.start ) );
-	report.data.set( 'fetchStart', normalize( getFetchStart(), report.start ) );
-	report.data.set( 'domainLookupStart', normalize( getDomainLookupStart(), report.start ) );
-	report.data.set( 'domainLookupEnd', normalize( getDomainLookupEnd(), report.start ) );
-	report.data.set( 'connectStart', normalize( getConnectStart(), report.start ) );
-	report.data.set( 'connectEnd', normalize( getConnectEnd(), report.start ) );
-	report.data.set( 'secureConnectionStart', normalize( getSecureConnectionStart(), report.start ) );
-	report.data.set( 'requestStart', normalize( getRequestStart(), report.start ) );
-	report.data.set( 'responseEnd', normalize( getResponseEnd(), report.start ) );
-	report.data.set( 'domLoading', normalize( getDomLoading(), report.start ) );
-	report.data.set( 'domInteractive', normalize( getDomInteractive(), report.start ) );
+	report.data.set( 'navigationStart', normalize( getNavigationStart(), report.beginning ) );
```

Comment:
Renamed `report.start` to `report.beginning` to be more consistent: now `start()/stop()` are methods, `beginning/end` are properties.